mindmeld-ui/yarn.lock

```diff
@@ -0,0 +1,8714 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Do we need to keep this file?

Reply:
Yes, this is so people can use either `npm` or `yarn` to manage the dependencies.